### ISSUE TITLE
macOS Monterey deleting bitcoin core after updates (M1)

### ISSUE THREAD

**Issue by nunya-mich** [bitcoin/bitcoin#23134]
Summary: When updating to macOS Monterey the bitcoin core app is deleted from the applications folder. If pinned to the dock you will see a "?" where it was in it's location.

Did not delete wallet or node data. Reinstalling the application works just fine.

**Comment by hebasto** [comment]
Did it happen to Bitcoin Core app only?
Is it a macOS Monterey expected or buggy behavior?

**Comment by nunya-mich** [comment]
Yes only to Bitcoin Core app while third party applications that have also been downloaded from an external source remain. This wasn't expected in prior builds but it has happened twice thus far on Beta 7 and 6 of macOS Monterey.

**Comment by hebasto** [comment]
@nunya-mich 

Is it self-compiled binary, or a signed release binary?

**Comment by nunya-mich** [comment]
Self-compiled is good to go it appears to be the signed release binary only at this time. The self-compiled survive the last beta update.

**Comment by TomasHaaris** [comment]
I have it too. This problem has existed for more than a year.
Such problem exists on Bigsur & Monterey.
I just installed bitcoin core and after tree days - no more program files on the computer.

Just these files:
<img width="752" alt="Screenshot 2022-01-23 at 14 54 27" src="https://user-images.githubusercontent.com/96632672/150679354-82b0f561-a66e-474e-8477-bb9432287c4a.png">
<img width="193" alt="Screenshot 2022-01-23 at 14 56 22" src="https://user-images.githubusercontent.com/96632672/150679456-9329a173-a059-4f12-90bb-7d13eb9c806b.png">

**Comment by coinforensics** [comment]
I was able to reproduce this behaviour in a virtual machine. 

After updating from macOS 12.2 to 12.2.1 the Bitcoin Core application `/Applications/Bitcoin-Qt.app/` was gone. Fortunately the data directory is still available.

XProtect does not have any related Yara rule and MRT only checks for something called "/QtBitcoinTrader.app". 

It must be related to the update process itself.

**Comment by coinforensics** [comment]
Same behavior from 12.2.1 to 12.3.
I really tried to contact Apple. So far no answer.
Nothing in the logs.

**Comment by hebasto** [comment]
https://github.com/bitcoin/bitcoin/issues/24501#issuecomment-1086857682

**Comment by procion1302** [comment]
Could confirm this bug exist

**Comment by hebasto** [comment]
@procion1302 
> Could confirm this bug exist

What Bitcoin Core version?

If not v23.0, please try it.

**Comment by VladimirBrejcha** [comment]
Still happens
